SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -7,15 +7,207 @@
 
 import SwiftUI
 
+/// Экран настроек
 struct ContentView: View {
+    
+    private enum Constants {
+        static let navigationTitleText = "Настройки"
+        static let bearImageName = "7"
+        static let airplaneModeImageName = "1"
+        static let wifiModeImageName = "2"
+        static let bluetoothImageName = "3"
+        static let CellularCommunicationImageName = "4"
+        static let modemImageName = "5"
+        static let vpnImageName = "6"
+        static let chevronForwardSystemImageName = "chevron.forward"
+        static let userName = "Beargreez"
+        static let profileInfoText = "Apple ID, iCloud, контент и покупки"
+        static let offersAppleIdText = "Предложения Apple ID"
+        static let offersAppleIdCount = "3"
+        static let updateIOSText = "iOS 16.2: уже доступно"
+        static let updateCount = "1"
+        static let airplaneModeText = "Авиарежим"
+        static let wiFiText = "Wi-Fi"
+        static let bluetoothText = "Bluetooth"
+        static let cellularCommunicationText = "Сотовая связь"
+        static let modemText = "Режим модема"
+        static let vPNText = "VPN"
+    
+        static let profileImageSide: CGFloat = 70
+        static let profileImageCornerRadius: CGFloat = 35
+        static let userNameShadowOffsetX: CGFloat = -40
+        static let userNameShadowOffsetY: CGFloat = 15
+        static let userFontSizeNumber: CGFloat = 25
+        static let userDescriptionFontSizeNumber: CGFloat = 12
+        static let userDescriptionShadowOffsetX: CGFloat = 10
+        static let minLengthSpacerUserContainer: CGFloat = 5
+        static let offersCornerRadius: CGFloat = 13
+        static let offersSideNumber: CGFloat = 25
+        static let iconCornerRadius: CGFloat = 10
+        static let zeroNumber = 0
+        
+        static let connectionStatuses = ["Подключен", "Не подключен"]
+        static let switchedStatuses = ["Вкл.", "Выкл."]
+        static let cellularCommunicationStatuses = ["Beeline", "MTS", "Megafhone"]
+    }
+    
+    @State private var isOnAirplaneMode = false
```

Comment:
```suggestion
    @State private var isOnAirplaneMode = false
```
все приватки ниже var body